home-work-angular/app-cat/votecat/votecat.service.js

```diff
@@ -1,15 +1,18 @@
 ﻿(function (module) {
-    function voteCatService($http, $q, $location) {
+    function voteCatService($http, $q, $location, localStorage) {
         var service = {
             GetCats: GetCats,
-            AddCat:AddCat,
-            GetCat:GetCat
+            AddCat: AddCat,
+            GetCat: GetCat,
+            AuthenticateUser: AuthenticateUser,
```

Comment:
**AuthenticateUserPromise** is already added to the service object, why this **AuthenticateUser** is again exposed